test/unit/article/article.spec.js

```diff
@@ -284,12 +302,6 @@ describe('Test for article', () => {
         expect(response.body.message).to.be.deep.equals('All articles found successfully');
       });
 
```

Comment:
Also, I would like to know why the test here was removed.

Reply:
I removed it as a result of conflict after I rebase staging into my branch. I noticed it was a repeated test on different lines.

Reply:
OK, looks good now. Good job!